.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "riscv-binutils-gdb"]
+	path = riscv-binutils-gdb
+	url = https://github.com/riscv/riscv-binutils-gdb.git
```

Comment:
What about using a relative submodule path?  People with clones of the entire toolchain will have an easier time working with it.